proxmox/config_qemu.go

```diff
@@ -40,6 +40,7 @@ type ConfigQemu struct {
 	Description     *string         `json:"description,omitempty"`
 	Disks           *QemuStorages   `json:"disks,omitempty"`
 	EFIDisk         QemuDevice      `json:"efidisk,omitempty"`   // TODO should be a struct
+	Storage         string          `json:"storage,omitempty"`   // for clone
```

Comment:
Could you update this comment to make it clear it only has effect when creating a full clone and this value is never returned.

Reply:
For sure, that's a good idea.